inst/metadata/advanced_run_options.json

```diff
@@ -214,6 +214,36 @@
 					"helpText": "t_(OPTIONS_ADVANCED_LOGIT_HELP)"
 				}
 			]
+		},
+	        {
+			"label": "Approximate district survey design effect",
+                       "helpText": "Approximate design effect to scale effective sample size for survey observations. Typically this will be smaller than the full survey design effect.",
+		        "controls": [
+				{
+					"name": "deff_prev",
+					"label": "Prevalence",
+					"type": "number",
+					"required": true,
+				        "value" = 1.0
```

Comment:
Should be 
```
"value": 1.0
```

Reply:
Sigh, thanks. This makes a strong case for just going to bed.

Fixed in 75406ff